app/controllers/graders_controller.rb

```diff
@@ -280,7 +280,7 @@ def criteria_with_assoc(assignment, options = {})
     criterion_ids = options[:criterion_ids]
     includes = options[:includes] || CRITERION_ASSOC
 
-    criteria = assignment.get_criteria.includes(includes)
+    criteria = assignment.get_criteria.empty? ? assignment.get_criteria : assignment.get_criteria.includes(includes)
```

Comment:
One of the test cases causes assignment.get_criteria to be empty. The includes method raises an error when applied to an empty array. That is why I added this check.